### PR TITLE
Update register-devices-partner.md

### PR DESCRIPTION
**Pull request by msbemba**
corrected the column order as it is Device serial number, Manufacturer name, and Device model

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3018

**Comment by jaimeo** [comment]
Do not make changes like this without checking the SME. Maybe it's the form that needs to change and not the doc....

**Comment by jaimeo** [comment]
@AlexBlain should the article change or is the download form out of date?

**Comment by msbemba** [comment]
@jaimeo Thank you. I'll keep that in mind.

**Comment by AlexBlain** [comment]
@jaimeo @msbemba this is actually from the Autopilot documentation : https://docs.microsoft.com/en-us/partner-center/autopilot#add-devices-to-a-customers-account
I did not change anything from what is written over there :)

File: microsoft-365/managed-desktop/get-started/register-devices-partner.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Once you have established the relationship with your customers, you can leverage
...
 
-If you get an error message while trying to upload the .csv file, check the format of the file. You can use the hardware hash only, or the OEM name, serial number, and model (in that column order), or the Windows Product ID. You can also use the sample .csv file provided from the link next to **Add devices** to create a device list. 
+If you get an error message while trying to upload the .csv file, check the format of the file. You can use the hardware hash only, or  serial number, Manufacturer name, and model (in that column order), or the Windows Product ID. You can also use the sample .csv file provided from the link next to **Add devices** to create a device list.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/register-devices-partner.md]:
```suggestion
If you get an error message while trying to upload the .csv file, check the format of the file. You can use the hardware hash only, or serial number, manufacturer's name, and model (in that column order), or the Windows product ID. You can also use the sample .csv file provided from the link next to **Add devices** to create a device list. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by jaimeo** [comment]
Per @AlexBlain, the current article matches the current Autopilot article. So either that article ALSO needs to have the order changed or neither of them does. @msbemba where are you getting the order you are proposing in your edit?

**Comment by msbemba** [comment]
@jaimeo I downloaded it from the partner center.

**Comment by msbemba** [comment]
> Per @AlexBlain, the current article matches the current Autopilot article. So either that article ALSO needs to have the order changed or neither of them does. @msbemba where are you getting the order you are proposing in your edit?

the downloaded file looks like this 

image

**Pull request merged**